invenio_records_files/__init__.py

```diff
@@ -172,9 +177,10 @@
 
 Integration with invenio-records-ui
 -----------------------------------
-If you are using ``invenio-records-ui``, you can easily add new views by
```

Comment:
In line `178`: Let's use the capitalised version of Invenio-Records-UI in the section title.